cscs-checks/microbenchmarks/osu/osu_tests.py

```diff
@@ -111,10 +111,10 @@ def __init__(self, variant):
                     'latency': (6.0, None, 0.10, 'us')
                 },
                 'daint:gpu': {
-                    'latency': (7.81, None, 0.25, 'us')
+                    'latency': (13.75, None, 0.25, 'us')
```

Comment:
The range that covers ~90% of the values is more or less 6us to 15us, with the most repeated values around 6-7us. I would put the reference at the most repeated value (~6.5us), but in that case the max would have to be close to 200%. Would that be an option?